common/src/test/java/io/druid/common/utils/ServletResourceUtilsTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.common.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ServletResourceUtilsTest
+{
+
+  @Test
+  public void testSanitizeException() throws Exception
+  {
+    final String message = "some message";
+    Assert.assertEquals(message, ServletResourceUtils.sanitizeException(new Throwable(message)).get("error"));
+    Assert.assertEquals("null", ServletResourceUtils.sanitizeException(null).get("error"));
+    Assert.assertEquals(message, ServletResourceUtils.sanitizeException(new Throwable()
```

Comment:
A throwable's toString is not guaranteed to be equal to its message. It can sometimes have more information. See `org.jets3t.service.ServiceException#toString` for an example.

At a minimum, if a throwable class doesn't implement a toString, and doesn't have a message, then it should at least print out something like `SomeCrazyException@78a31d` 

I could have tested the throwable's toString before the call, and use that as the expected value. But I thought it was easier in this test to explicitly specify the toString result, and verify against that.